symbolic-il2cpp/src/usym.rs

```diff
@@ -416,19 +451,25 @@ mod tests {
         header.os = push_string(usyms.get_string(header.os as usize).unwrap()) as u32;
         header.arch = push_string(usyms.get_string(header.arch as usize).unwrap()) as u32;
 
-        // Construct new records.
-        header.record_count = 5;
+        // Construct new records. Skims the top 5 records, then grabs the 3 records that have
+        // mappings to managed symbols.
+        header.record_count = 5 + 3;
+        let first_five = usyms.records.iter().take(5);
+        let actual_mappings = usyms.records.iter().filter(|r| r.managed_symbol != 0);
```

Comment:
I think this also relies on the empty string living at offset 0, again I don't really trust this.